### ISSUE TITLE
Create wiki article about mappings configuration and how to support mappings from other ORM/custom mapping sources

### ISSUE THREAD

**Issue by MaceWindu** [linq2db/linq2db#904]

**Comment by max-arshinov** [comment]
Hi. Is there a way to avoid using [Table] and [Column] attributes? I am wondering if I can use same POCO classes with EF and linq 2 db. A lot of code is written with EF. If I can reuse it's mapping I can refactor code from EF to linq 2 db step by step.

**Comment by sdanyliv** [comment]
Yes, check this discussion thread https://github.com/linq2db/linq2db/issues/900

**Comment by max-arshinov** [comment]
Thank you. So to support Data Annotation Attributes I need to create MappingSchema and then use reflection to build entity mappings using GetFluentMappingBuilder().Entity<>..., right?

**Comment by sdanyliv** [comment]
Not exactly,  based on discussion thread you can convert current attributes to our attributes. But yes you need to define mapping schema, for example in DataConnection constructor.

**Comment by MaceWindu** [comment]
Closing for now...
